tests/idris2/mkdoc/mkdoc001/run

```diff
@@ -0,0 +1,4 @@
+. ../../../testutils.sh
+
+idris2 --mkdoc test.ipkg
+head -n -1 build/docs/index.html
```

Comment:
I'd suggest to add a small filter to remove the header from the `expected` file:

```suggestion
head -n -1 build/docs/index.html | sed '1,/<\/header>/d'
```